pkg/utils/nodeclaim/nodeclaim.go

```diff
@@ -474,7 +478,7 @@ func IsExpired(obj client.Object, clock clock.Clock, nodePool *v1beta1.NodePool)
 }
 
 func GetExpirationTime(obj client.Object, nodePool *v1beta1.NodePool) time.Time {
-	if nodePool == nil || nodePool.Spec.Deprovisioning.ExpirationTTL.Duration >= 0 || obj == nil {
+	if nodePool == nil || nodePool.Spec.Deprovisioning.ExpirationTTL.Duration < 0 || obj == nil {
```

Comment:
?

Reply:
Removed this function and are doing the check in the the expiration disruption directly